Resources/config/admin.xml

```diff
@@ -27,6 +27,7 @@
             <argument />
             <argument>%sonata.classification.admin.tag.entity%</argument>
             <argument>%sonata.classification.admin.tag.controller%</argument>
+            <argument type="service" id="sonata.classification.manager.context" />
```

Comment:
The first parameters are strings, the fourth is an object. So we can't use any variable.